src/platform/mhv/api/mocks/index.js

```diff
@@ -158,4 +170,4 @@ const responses = {
   },
 };
 
-module.exports = delay(responses, 1000);
+module.exports = delay(responses, 3000);
```

Comment:
by those that use these mocks with Codespaces, it's been asked to not extend this delay. Is there a chance we can keep it at 500-1000 range? Or is there a specific reason why we would want to increase to 3000?

Reply:
I like using 3000 to put the UI to the test of the longer response times. I will lower this back for the codespaces